src/ui/Form.tsx

```diff
@@ -1,2 +1,20 @@
 import { styled } from 'stitches.config';
-export const Form = styled('form', {});
+export const Form = styled('form', {
+  variants: {
+    nominate: {
+      true: {
+        position: 'relative',
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        backgroundColor: 'white',
+        width: '100%',
+        borderRadius: 8,
+        padding: '0 0 $lg',
+        overflowY: 'auto',
+        maxHeight: '100vh',
+        maxWidth: '648px',
```

Comment:
you should be able to avoid hardcoding a value like this, and instead rely on the fact that `Modal` already has a maxWidth set.